pyposeidon/utils/mplot.py

```diff
@@ -335,19 +359,22 @@ def grid(self, **kwargs):
 
         if coast is not None:
             try:
-                del kwargs["coastlines", "R"]
+                del kwargs["coastlines"]
+            except:
+                pass
+            try:
+                del kwargs["R"]
```

Comment:
Please avoid `except: pass`. It is almost never a good idea.
In this case there are two options:

``` python
for name in ("coastlines", "R"):
    if name in kwargs:
        del kwargs[name]
``` 
or using the `dict.pop()` method:
``` python
for name in ("coastlines", "R"):
    kwargs.pop(name, None)
```

https://docs.python.org/3/library/stdtypes.html#dict.pop

Reply:
You are right. I will replace it in the next PR. These files will probably change during the tests for the Tutorial.